airflow/operators/docker_operator.py

```diff
@@ -231,8 +232,8 @@ def execute(self, context):
                 self.log.info(line)
 
             exit_code = self.cli.wait(self.container['Id'])
-            if exit_code != 0:
-                raise AirflowException('docker container failed')
+            if exit_code['StatusCode'] != 0:
```

Comment:
Thanks @ash! Looks good to me.

A suggestion: given the “self.cli.wait” returns “full” response as a dict now, shall we change the variable name as well? Say from “exit_code” to something like “cli_response” or “cli_result”? Otherwise it may be confusing later.

Reply:
Good call.